pages/auth/login.vue

```diff
@@ -322,11 +323,11 @@ export default {
                 />
               </div>
               <div class="">
-                <LabeledInput
+                <Password
                   ref="password"
-                  v-model="password"
-                  type="password"
+                  v-model.trim="password"
                   :label="t('login.password')"
+                  :required="true"
```

Comment:
For reasons that I can't remember we don't show the `required` indicator on either username or password, so this should be removed

Reply:
Done